dspace-server-webapp/src/main/java/org/dspace/app/rest/security/jwt/JWTTokenRestAuthenticationServiceImpl.java

```diff
@@ -47,9 +48,13 @@
     private static final String AUTHORIZATION_COOKIE = "Authorization-cookie";
     private static final String AUTHORIZATION_HEADER = "Authorization";
     private static final String AUTHORIZATION_TYPE = "Bearer";
+    private static final String AUTHORIZATION_TOKEN_PARAMETER = "token";
 
     @Autowired
-    private JWTTokenHandler jwtTokenHandler;
+    private SessionJWTTokenHandler sessionJWTTokenHandler;
```

Comment:
@KevinVdV : I agree with @abollini  here. The word "session" is not really appropriate here as it's confusing (is this a user session, an HTTP session, a Hibernate Session, or something else?).  I'd prefer we just call this `LoginJWTTokenHandler` as he suggested.

Similarly, all the new configurations should be `jwt.login.*` instead of `jwt.session.*` and similar.